tuned/units/manager.py

```diff
@@ -18,14 +19,15 @@ class Manager(object):
 	"""
 
 	def __init__(self, plugins_repository, monitors_repository,
-			def_instance_priority, hardware_inventory):
+			def_instance_priority, hardware_inventory, config = None):
 		super(Manager, self).__init__()
 		self._plugins_repository = plugins_repository
 		self._monitors_repository = monitors_repository
 		self._def_instance_priority = def_instance_priority
 		self._hardware_inventory = hardware_inventory
 		self._instances = []
 		self._plugins = []
+		self._config = GlobalConfig() if config is None else config
```

Comment:
I recently realized that there's a shortcut for this. Use whichever you prefer. (Note that it's not entirely semantically equivalent, because there are other values that evaluate to False, but for our purposes it works.)
```
		self._config = config or GlobalConfig()
```